how-to.md

```diff
@@ -27,6 +27,23 @@
 
 You can verify the licenses in your tenant [in the Azure Portal](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Licenses) under **All products**
 
+## Tenant install options
+
+* **Delegated Authentication**: This option allows Simeon to read and write to the tenant with a user of your choosing (typically a pre-existing Global Administrator in the tenant or a user with the minimum [required permissions](https://simeoncloud.github.io/docs/#/permissions)). Simeon recommends using delegated authentication for all production tenants.
+  * Simeon will create a refresh token for the user you authenticate with. This refresh token will cache the sign-in information for Simeon to use.
+  * You can authenticate with a user subject to MFA, Conditional Access, and other security policies.
+  * If you change the login credentials, MFA, or other sign-in policies for that user, you will need to log into the Simeon app and [reauthenticate](https://simeoncloud.github.io/docs/#/how-to?id=re-prompt-sync-to-complete-delegated-authentication) to create a new refresh token.
```

Comment:
Change this to

If the refresh token is invalidated, then you will need to log into the Simeon app and [reauthenticate](https://simeoncloud.github.io/docs/#/how-to?id=re-prompt-sync-to-complete-delegated-authentication) to create a new refresh token. The token can be invalidated for reasons including changes to the login credentials, re-requiring MFA, and token expiration policies.